connectors/odata/translator-odata4/pom.xml

```diff
@@ -54,10 +54,6 @@
 		    <groupId>org.apache.commons</groupId>
 		    <artifactId>commons-lang3</artifactId>
 		</dependency>
-		<dependency>
```

Comment:
slf4j is a transitive dependency of olingo.  Does it seem to be optional?

Reply:
Not optional, if an exclusion is added, a ClassNotFoundException for the LoggerFactory occurs.

Reply:
I don't quite follow the response.  It should be declared if it's actually being used - we don't want to rely on implicit transitive dependencies.